pkgs/applications/emulators/xemu/default.nix

```diff
@@ -0,0 +1,132 @@
+{ stdenv
+, fetchFromGitHub
+, lib
+, qemu
```

Comment:
Where is qemu used (besides the licensing)?

Reply:
Sorry, thats a leftover from trying to get it to build at all. I assume I should remove it?

Reply:
Yes, please.

Reply:
Done